### DISCUSSION TITLE
Import TIGER data after freeze DB

### DISCUSSION THREAD

**Discussion by nikitadtastewise** [osm-search/Nominatim#3048] [Q&A] [answered]
What are the steps to upload TIGER data after DB has been freezed?

I used Helm chart solution to import North America OSM data. It has been successfully imported but according to InitJob, it applied `nominatim freeze` after the import. I run import of TIGER data but do not see results. Could you give me some SQL command to verify table of tiger data?

**Comment by mtmail** [comment]
You should see this in your database:
```SQL
select count(*) from location_property_tiger;
 33842890
```
Make sure you've set the environment variable and that the website files got regenerated.
https://nominatim.org/release-docs/latest/customize/Tiger/

Try a query like https://nominatim.openstreetmap.org/reverse.php?lat=38.65521&lon=-92.12890&zoom=18&format=jsonv2&debug=1 It should print "Tiger house number result" and show several SQL queries looking at the "location_property_tiger" table.

**Reply by nikitadtastewise** [reply]
@mtmail Seems I don't have all data. 
```

nominatim=> select count(*) from location_property_tiger;
 count
--------
 197866
(1 row)
```
What is the right way to upload TIGER data again?

I'm planning to do the following:
1) Mount flatnode.file for pod in k8s.
2) sed -i 's\^NOMINATIM_FLATNODE_FILE=$\NOMINATIM_FLATNODE_FILE="/nominatim/flatnode/flatnode.file"\g' /nominatim/.env 
3) nominatim replication --init
4) nominatim add-data --tiger-data tiger-nominatim-preprocessed-latest.csv.tar.gz
5) echo NOMINATIM_USE_US_TIGER_DATA=yes >> .env
6) nominatim refresh --functions --website
7) nominatim freeze

How much time usually it should take?

**Reply by mtmail** [reply]
I'm not familiar enough with k8s but the commands seem correct. Expect about 90minutes for `nominatim add-data --tiger-data`. The second last output will be "Creating indexes on Tiger data", index creation is only 1 minute.

**Comment by lonvia** [comment]
See #2463. Importing Tiger data after a `nominatim freeze` was run is currently not possible.

**Reply by nikitadtastewise** [reply]
Are you 100% sure? because re-import North America is a pain...... 
Does it mean that `nominatim freeze` should not be applied to be able to update TIGER data in future too?

**Reply by lonvia** [reply, marked as answer]
Yes, I  am 100% sure. As long as #2463 is not implemented, you must keep the update tables if you want to add TIGER data.
